spec/plot_spec.rb

```diff
@@ -67,23 +91,44 @@
     context 'Googlecharts library' do
       before { Daru::View.plotting_library = :googlecharts }
       let(:lib) { Daru::View.plotting_library }
+      let(:options) {{width: 800, height: 720}}
```

Comment:
`{{width: 800, height: 720}}` -> `{width: 800, height: 720}`

Reply:
I tried doing it, but am getting the following Syntax error: `syntax error, unexpected ':', expecting '}'
        let(:options) {width: 800, height: 720}`
and as mentioned here: https://stackoverflow.com/questions/21336438/using-a-hash-inside-a-let, double brackets should be used.

Reply:
Ah!  it's correct, since we are using let.